README.md

```diff
@@ -54,28 +56,41 @@ def f_0():
 
 f_0()
 ```
-is a `python` program generated with [`string_to_code.to_python3`](./string_to_code/to_python3.py) [displaying](https://www.online-python.com/jgzNiCAvxR) `Hello, World!`.
 
+is a `python` program generated with
+[`string_to_code.to_python3`](./string_to_code/to_python3.py)
+[displaying](https://www.online-python.com/jgzNiCAvxR) `Hello, World!`.
 
 In order to generate a code in `your_favourite_language` just call the function `string_to_code.to_your_favourite_language.proc` with the string which you want to display. [`examples`](./examples) show some basic usage.
 
-# Adding a new language
+## Adding a new language
+
+If you think that [`string_to_code`](./string_to_code) is missing some
+language, feel free to add it - it is simple.
+If you want to add support of let's say `turbo_snake`,
+you will need to add three files:
+
+  - `to_turbo_snake.py` into [`string_to_code`](./string_to_code).
+    This module should contain a function `proc` taking two arguments:
+    - `in_str` being the string which the generated code should display,
+    - `gen_function_names=None` being a generator which yields the function names in generated code.
```

Comment:
Codacy has a fix for the issue: [Expected: 2; Actual: 4](https://app.codacy.com/gh/vil02/string_to_code_proj/pullRequest?prid=9915707)
```suggestion
  - `gen_function_names=None` being a generator which yields the function names in generated code.
```